doc/md/schema-edges.md

```diff
@@ -932,3 +949,8 @@ func (Pet) Edges() []ent.Edge {
 ```
 
 Read more about annotations and their usage in templates in the [template doc](templates.md#annotations).
+
+## Naming Convention
+
+By convention edge names should use `snake_case`. The corresponding struct fields generated by `ent` will follow the Go convention
+of using `camelCase`. In cases where `camelCase` is desired, you can do so with the `StorageKey` or `StructTag` methods.
```

Comment:
I think it should be `PascalCase` because struct fields are exported. Right? 

```suggestion
of using `PascalCase`. In cases where `PascalCase` is desired, you can do so with the `StorageKey` or `StructTag` methods.
```

Reply:
🤦  yep, good point. Done.